Lib/unittest/async_case.py

```diff
@@ -161,3 +161,12 @@ def run(self, result=None):
             return super().run(result)
         finally:
             self._tearDownAsyncioLoop()
+
+    def debug(self):
+        self._setupAsyncioLoop()
+        super().debug()
+        self._tearDownAsyncioLoop()
+
+    def __del__(self):
+        if self._asyncioTestLoop is not None:
+            self._tearDownAsyncioLoop()
```

Comment:
Looks a little weird but I don't have better idea.

Reply:
Yes, it looks very weird. And you should ensure that the IsolatedAsyncioTestCase instance is destroyed before any other event loop is run, otherwise you can get an ugly message on the stderr during garbage collecting.